packages/plugin-terraform/test/testdata/base.graphql

```diff
@@ -31,6 +31,10 @@ type TestNoAutoSync @amplienceContentType {
   name: String
 }
 
+type TestCustomLabel @amplienceContentType(label: "Test Custom Label") {
```

Comment:
This is a bit of a not telling testcase, since the generated fallback label will also be "Test Custom Label".

Reply:
Good point, two ticks...

Reply:
All done 👍 Ready for re-review.